pkg/tags/tags.go

```diff
@@ -143,3 +156,31 @@ func (ts *Tags) UnmarshalJSON(value []byte) error {
 
 	return err
 }
+
+// GetTagFromStore get a given tag from the state store.
+func (ts *Tags) GetTagFromStore(uid uint32) (*Tag, error) {
```

Comment:
This method does not have to be exported. The correct way to get the tag from outside of the package is by the Get method.